### ISSUE TITLE
Trying to understand the training on LSUN dataset with multi-class labels

### ISSUE THREAD

**Issue by cardyfib** [nashory/pggan-pytorch#36]
Hi @nashory 
Thanks a lot for this implementation! I am trying to use PG-GAN for multi-class dataset (such as LSUN).

I have a question, and I was wondering if I can get your thoughts: The paper mentions that their training is unsupervised - meaning that it was not label-conditioned. Then how come they were able to generate label-specific images for LSUN dataset? Did they train separate networks for each label or is their network a multi-class generator?

Any information on multi-class PGGAN training will be of great help.
Thanks in advance!

**Comment by cardyfib** [comment]
I found that they trained separate network for each label. So the PG-GAN in the original paper is not multi-class.
